modxcms/revolution
# Restore ability to render multi-select listbox TV values by selection order

### What does it do?
Adds a new input option for this TV type to control the sorting of saved values when rendered to a Resource.

### Why is it needed?
A community member requested this functionality be brought back, pointing to possible use cases where preservation of the order of selections made is desirable.

### How to test
Create a Listbox (Multiple-Select) TV and experiment with changing the TV behavior via the new Preserve Entry Order option and verify that when this option is set to “Yes,” randomly selected options are shown in the order selected after saving and reloading. Conversely, verify the current default behavior remains intact (i.e., saved values appear in the same order defined by the list itself, no matter the order in which they were selected).

### Related issue(s)/PR(s)
Resolves #16495.
